themes/openy_themes/openy_lily/sass/modules/_google_search.scss

```diff
@@ -3,6 +3,16 @@
  */
 
 .viewport {
```

Comment:
I don't think we need this wrapper here. We can use `.paragraph--type--google-search`
The same should apply across all themes.